### DISCUSSION TITLE
New projects section on front page

### DISCUSSION THREAD

**Discussion by pikachub2005** [ScratchAddons/ScratchAddons#3597] [Ideas] [answered]
https://api.scratch.mit.edu/proxy/featured contains a list of new projects under "community_newest_projects." I thought this would be nice at the bottom, so it's not in the way of more important stuff. It would also allow newer projects to get more views.

**Comment by Dhuls-Scratch** [comment, marked as answer]
Would it really though? Scratch Addons has around only 30,00 users. Plus, it was removed by the ST because of the possibility it showed inappropriate stuff, and how projects barely stayed there.

**Comment by TheColaber** [comment]
Isn't that what the resent filter shows? Or am I wrong?
